bundles/org.eclipse.swt/Eclipse SWT Browser/common/org/eclipse/swt/browser/WebBrowser.java

```diff
@@ -36,6 +37,7 @@ abstract class WebBrowser {
 
 	static final String ERROR_ID = "org.eclipse.swt.browser.error"; // $NON-NLS-1$
 	static final String EXECUTE_ID = "SWTExecuteTemporaryFunction"; // $NON-NLS-1$
+	private static final URI BASE_URI = URI.create("about:blank");
```

Comment:
Please change this to a more self-explaining name, such as `URI_FOR_CUSTOM_TEXT_PAGE`.

Reply:
Done